src/app/pages/quickMaps/components/quickMapsHeader.component/quickMapsHeader.component.scss

```diff
@@ -34,7 +34,7 @@
   margin-left: 2%;
 }
 
-.qmTitle {
+.qm-title {
```

Comment:
case conventions! 
![image](https://user-images.githubusercontent.com/4210022/116383834-2396e700-a80f-11eb-8a9b-486a95607a7e.png)

Reply:
In EPOS sonarQube was used to do an evaluation of the GUI.  This kinda thing came up loads.  I think kabab-case is standard in css.  Probably one of the more standard standards!